docs/tutorial/vald-multicluster-on-k8s.md

```diff
@@ -0,0 +1,290 @@
+     vald-mirror-gateway-6df75cf7cf-gzcr4   1/1     Running   0          2m26s
+     vald-mirror-gateway-6df75cf7cf-vjbqx   1/1     Running   0          2m26s
+     vald-mirror-gateway-6df75cf7cf-c2g7t   1/1     Running   0          2m41s
+     ```
+
+   - Output for `vald-02`
+
+     ```bash
+     kubectl get pods -n vald-02
+     NAME                                  READY   STATUS    RESTARTS   AGE
+     vald-agent-ngt-0                      1/1     Running   0          2m52s
+     vald-agent-ngt-1                      1/1     Running   0          2m52s
+     vald-agent-ngt-2                      1/1     Running   0          2m52s
+     vald-agent-ngt-4                      1/1     Running   0          2m52s
+     vald-agent-ngt-5                      1/1     Running   0          2m52s
+     vald-agent-ngt-3                      1/1     Running   0          2m52s
+     vald-discoverer-8cfcff76-vlmpg        1/1     Running   0          2m52s
+     vald-lb-gateway-54896f9f49-wtlcv      1/1     Running   0          2m52s
+     vald-lb-gateway-54896f9f49-hbklj      1/1     Running   0          2m52s
+     vald-manager-index-676855f8d7-bb4wb   1/1     Running   0          2m52s
+     vald-lb-gateway-54896f9f49-kgrdf      1/1     Running   0          2m52s
+     vald-mirror-gateway-6598cf957-t2nz4   1/1     Running   0          2m37s
+     vald-mirror-gateway-6598cf957-wr448   1/1     Running   0          2m52s
+     vald-mirror-gateway-6598cf957-jdd6q   1/1     Running   0          2m37s
+     ```
+
+   - Output for `vald-03`
+
+     ```bash
+     kubectl get pods -n vald-03
+     NAME                                   READY   STATUS    RESTARTS   AGE
+     vald-agent-ngt-0                       1/1     Running   0          2m46s
+     vald-agent-ngt-1                       1/1     Running   0          2m46s
+     vald-agent-ngt-2                       1/1     Running   0          2m46s
+     vald-agent-ngt-3                       1/1     Running   0          2m46s
+     vald-agent-ngt-4                       1/1     Running   0          2m46s
+     vald-agent-ngt-5                       1/1     Running   0          2m46s
+     vald-discoverer-879867b44-8m59h        1/1     Running   0          2m46s
+     vald-lb-gateway-6c8c6b468d-ghlpx       1/1     Running   0          2m46s
+     vald-lb-gateway-6c8c6b468d-rt688       1/1     Running   0          2m46s
+     vald-lb-gateway-6c8c6b468d-jq7pl       1/1     Running   0          2m46s
+     vald-manager-index-5596f89644-xfv4t    1/1     Running   0          2m46s
+     vald-mirror-gateway-7b95956f8b-l57jz   1/1     Running   0          2m31s
+     vald-mirror-gateway-7b95956f8b-xd9n5   1/1     Running   0          2m46s
+     vald-mirror-gateway-7b95956f8b-dnxbb   1/1     Running   0          2m31s
+     ```
+
+## Deploy ValdMirrorTarget Resource (Custom Resource)
+
+It requires applying the `ValdMirrorTarget` Custom Resource to the one namespace.
```

Comment:
You use both `namespace` and `Namespace` in this document.
Which is correct?

Reply:
I think `namespace` is correct. so I will unify to namespace 🙏